client/src/components/Home.js

```diff
@@ -1,8 +1,13 @@
-import React from 'react';
+import React, {Fragment} from 'react';
 import { Header, } from 'semantic-ui-react';
+import WhatDo1 from './WhatDo1'
 
 const Home = () => (
+  <Fragment>
```

Comment:
Indent the code in the Fragment. Let's also get right of the <Fragment> up in the import and just use the <> syntax. What you have isn't wrong but this way it will be more up to date